### PR TITLE
Updated to allow PHP 7.2

### PR DESCRIPTION
**Pull request by iainmckay**
Updates the version of `prooph/php-cs-fixer-config` to allow installation against PHP 7.2.

### CODE REVIEW

**Comment by prolic** [comment]
Can you please add it to travis build matrix as well?

**Comment by iainmckay** [comment]
Sure, done

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,14 @@ matrix:
     - php: 7.1
       env:
         - DEPENDENCIES="--prefer-lowest --prefer-stable"
+    - php: 7.2
+      env:
+        - DEPENDENCIES=""
+        - EXECUTE_CS_CHECK=true
```
Comment (prolic) [inline .travis.yml]: remove this line please, EXECUTE_CS_CHECK is meant only to be executed once per build

Comment (iainmckay) [reply .travis.yml]: Done

**Review by prolic — APPROVED** [review]

**Pull request merged**